common-ktx/src/test/java/co/nimblehq/common/extensions/DateExtensionsTest.kt

```diff
@@ -0,0 +1,339 @@
+package co.nimblehq.common.extensions
+
+import org.junit.Assert.*
+import org.junit.Test
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * Unit test for DateExtensions.kt.
+ */
```

Comment:
I think we could get rid of the comment on top class, it's a bit redundant IMO, the class name already shows the purpose of this class. Please check the all test classes on this PR for updating.